docs/release-notes/highlights-2.16.0.asciidoc

```diff
@@ -1,6 +1,13 @@
 [[release-highlights-2.16.0]]
 == 2.16.0 release highlights
 
+[float]
+[id="{p}-2160-known-issues"]
+=== Known issues
+
+- Enabling a Kibana hardened security context as well as enabling Kibana audit logs will cause Kibana to go into a `CrashLoopBackoff` state with the error `read-only file system, open '/usr/share/kibana/logs/audit.log'`.
```

Comment:
Since users do not set explicitly this security context, should we instead mention which Kibana versions[^1] are affected when upgrading to `2.16` ?

[^1]: I'm not sure myself actually, should it be `7.9` as mentioned [here](https://github.com/elastic/cloud-on-k8s/blob/0efeb943cd04bfaa252f2950a7fff410d40085c4/pkg/controller/kibana/pod.go#L146-L149), or `7.10` as tested [here](https://github.com/elastic/cloud-on-k8s/blob/0efeb943cd04bfaa252f2950a7fff410d40085c4/pkg/controller/kibana/initcontainer/configmap.go#L27-L28)?

Reply:
I think we should clean this up. It looks like 7.9 works after a quick smoke test I did with the security context in place with one error/warning:
```
Babel could not write cache to file: /usr/share/kibana/optimize/.babel_register_cache.json 
because it resides in a readonly filesystem. Cache is disabled.
```

Reply:
Fixed in https://github.com/elastic/cloud-on-k8s/pull/8413. Will update the wording.

Reply:
Updated in https://github.com/elastic/cloud-on-k8s/pull/8412/commits/40b51b3ce39895743f29712bef98c9331361357f